ui/server/src/main/scala/pl/touk/nussknacker/ui/util/WithDirectives.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.ui.util
+
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model.{HttpMethods, HttpResponse, StatusCodes}
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives.{complete, respondWithHeaders}
+
+object WithDirectives {
+  def apply(directives: Directive0*): Directive0 = directives.reduce((d1, d2) => d1.and(d2))
+}
+
+object WithHeaders {
```

Comment:
Maybe WithHeadersIfNotOptions or sth?

Reply:
Why? For request method `OPTIONS` we return those headers

Reply:
sorry, I misread it :)